### PR TITLE
Test changing MaxDynamicClusterSize and MinDynamicClusterSize using dynamic update

### PR DESCRIPTION
**Pull request by vanajamukkara**
Modify MaxDynamicClusterSize and MinDynamicClusterSize using dynamic update.
  Verify the cluster cannot be scaled beyond the modified MaxDynamicClusterSize value
  and cannot be scaled below MinDynamicClusterSize when allowReplicasBelowMinDynClusterSize is set false.
  Verify JMS message and connection distribution/load balance after scaling the cluster.

Jenkins results -
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4101/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4103/ (In progress)

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDynamicUpdate.java [hunk 1]
```diff
@@ -922,8 +929,8 @@ public void testOnNonDynamicChangesCancelUpdate() {
 
     // check that the domain status condition type is "OnlineUpdateCanceled" and message contains the expected msg
     String expectedMsgForCancelUpdate = "Online update completed successfully, but the changes require restart and "
-          + "the domain resource specified 'spec.configuration.model.onlineUpdate.onNonDynamicChanges=CancelUpdate' "
-          + "option to cancel all changes if restart require.";
+        + "the domain resource specified 'spec.configuration.model.onlineUpdate.onNonDynamicChanges=CancelUpdate' "
+        + "option to cancel all changes if restart require.";
```
Comment (jshum2479) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDynamicUpdate.java:933]: We don't have cancelUpdate anymore.  Do we still need this?

Comment (vanajamukkara) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDynamicUpdate.java:933]: Currently the test is disabled. If this feature is not going to be added in future releases I can delete this test. Please confirm.

Comment (jshum2479) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDynamicUpdate.java:933]: It's unlikely we will add this back soon, and if we indeed put it back then you can always use history to get the code back.

Comment (vanajamukkara) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDynamicUpdate.java]: ok, removed the test

**Review by anpanigr — APPROVED** [review]

**Review by xiancao — APPROVED** [review]
LGTM

**Review by jshum2479 — APPROVED** [review]

**Pull request merged**